radicle/src/cob.rs

```diff
@@ -0,0 +1,121 @@
+pub use cob::{
+    identity, object::collaboration::error, CollaborativeObject, Create, Entry, History, ObjectId,
+    TypeName, Update,
+};
+use radicle_cob as cob;
+use radicle_git_ext::Oid;
+
+use crate::{
+    identity::{project::Identity, Did},
+    storage::git::Repository,
+};
+
+/// The `Author` of a [`create`] or [`update`].
+///
+/// **Note**: `Author` implements [`identity::Identity`], but since it
+/// is not content-addressed, the [`identity::Identity::content_id`]
+/// returns [`git2::Oid::zero`]. This means that if the `author` is
+/// set in the updates the history entries for those changes will
+/// contain the zero `Oid` for the `author` field.
```

Comment:
I'm definitely missing some context here, let's chat about this at some point. I'd like to understand this `Identity` trait better and if we need to implement it given the caveat mentioned, or if there's a way to split it out or change the trait, eg. have content_id return an `Option<Oid>`.